src/metadata/persistence/avu.clj

```diff
@@ -1,52 +1,92 @@
 (ns metadata.persistence.avu
-  (:use [korma.core :exclude [update]]
-        [korma.db :only [transaction]]
-        [slingshot.slingshot :only [throw+]])
-  (:require [kameleon.db :as db]
-            [korma.core :as sql]))
-
-(defn- target-where-clause
-  "Adds a where-clause to the given query for the given target."
-  [query target-type target-id]
-  (where query {:target_id   target-id
-                :target_type (db/->enum-val target-type)}))
+  (:use [slingshot.slingshot :only [throw+]])
+  (:require [metadata.util.db :refer [ds t]]
+            [next.jdbc :as jdbc]
+            [next.jdbc.plan :as plan]
+            [next.jdbc.sql :as jsql]
+            [next.jdbc.types :as jtypes]
+            [honey.sql :as sql]
+            [honey.sql.helpers :as h]
+            [kameleon.db :as db]))
+
+(def avu-columns [:id :attribute :value :unit :target_id :target_type :created_by :modified_by :created_on :modified_on])
+
+(defn- avus-base-query
+  [cols]
+  (-> (apply h/select cols)
+      (h/from (t "avus"))))
```

Comment:
I've been doing a pattern like these two definitions in a bunch of places. Basically, have a plain `def` with the most common setup of columns, plus a `whatever-base-query` function that takes a list of columns and selects them from the appropriate table. I'm not sure if it's super useful, but it does cut down a little bit at least. May be worth generalizing, idk